### PR TITLE
Feature/initialize

### PR DESCRIPTION
**Pull request by perovsk1te**
I make crawl page .
I make other page only in design.
I will change state in crawl.

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+export const submittingUrl = (url) => {
+  return {
+    type: 'SUBMITTING_URL',
+    payload: {
+              url:url
+              }
+  };
+}
+
+export const incTruth = (id) => {
```
Comment (y-matsuwitter) [inline src/actions/index.js:19]: `voteTruth` would be better than `inc` as this action will be displayed as a "Vote".

File: src/actions/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  return{
+    type: 'INC_TRUTH',
+    payload: {
+              id:id
+              }
+  }
+}
+
+export const incFake = (id) => {
```
Comment (y-matsuwitter) [inline src/actions/index.js:28]: Same as above. You should use `vote`.

File: src/components/Crawl/List.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { List } from 'antd';
+import React, { Component } from 'react'
+
+export default class UrlList extends Component{
```
Comment (y-matsuwitter) [inline src/components/Crawl/List.js:4]: This file name should be `UrlList`. It's better to use the same name as a default class name.

File: src/components/Crawl/List.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  render(){
+    const data=this.props.list
+    return(
+      <div>
+        <List
+          size="small"
+          bordered
+          dataSource={data}
+          renderItem={item => (
+            <List.Item>
```
Comment (y-matsuwitter) [inline src/components/Crawl/List.js:14]: I recommend you to separate the list items component to another file.

File: src/components/Crawl/SubmitModal.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import { Modal, Button } from 'antd';
+import React from 'react';
+
+class SubmitModal extends React.Component {
+  state = { visible: false }
+
+  showModal = () => {
+    this.setState({
```
Comment (y-matsuwitter) [inline src/components/Crawl/SubmitModal.js:8]: Do not use setState in the component class if you use react-redux. Please put this logic to mapStateToProps & mapDispatchToProps.

File: src/components/Crawl/SubmitModal.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+      visible: true,
+    });
+  }
+
+  handleOk = (e) => {
+    console.log(e);
+    this.setState({
```
Comment (y-matsuwitter) [inline src/components/Crawl/SubmitModal.js:15]: Same as above.

File: src/components/Crawl/SubmitModal.js [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+      visible: false,
+    });
+    return this.props.onClick();
+  }
+
+  handleCancel = (e) => {
+    console.log(e);
+    this.setState({
```
Comment (y-matsuwitter) [inline src/components/Crawl/SubmitModal.js:23]: Same as above.

File: src/components/Sidebar.js [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+const { Sider } = Layout
+
+class Sidebar extends React.Component {
+  state = {
+    collapsed: false
+  }
+
+  onCollapse = collapsed => {
+    this.setState({ collapsed })
```
Comment (y-matsuwitter) [inline src/components/Sidebar.js:13]: Same as I mentioned above.

File: src/components/Sidebar.js [hunk 9]
```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Layout, Menu, Icon } from 'antd'
+
+const { Sider } = Layout
+
+class Sidebar extends React.Component {
+  state = {
```
Comment (y-matsuwitter) [inline src/components/Sidebar.js:8]: Initial state should be defined at `createStore`.

File: src/components/Vote/VoteModal.js [hunk 10]
```diff
@@ -0,0 +1,51 @@
+import { Modal, Button } from 'antd';
+import React from 'react';
+
+class VoteModal extends React.Component {
+  state = { visible: false }
+
+  showModal = () => {
+    this.setState({
```
Comment (y-matsuwitter) [inline src/components/Vote/VoteModal.js:8]: Same as I mentioned above.

**Review by y-matsuwitter — CHANGES_REQUESTED** [review]
Though not yet finished, please check my comments especially about `react-redux` usage and variables naming.

File: src/components/Vote/VoteModal.js (hunk 10 above)
Comment (perovsk1te) [reply src/components/Vote/VoteModal.js:8]: Thank you for your comment.

**Pull request closed without merging**